spec/factories/transactions.rb

```diff
@@ -2,6 +2,13 @@
   factory :transaction do
     user
     amount { Faker::Number.between(100, 1000) }
-    transactinable { |transaction| transaction.association(:balance_transaction) }
+
+    trait :balance_trans do
+      transactinable { |transaction| transaction.association(:balance_transaction) }
+    end
```

Comment:
@marderer `trans`? 😄 
I guess there is word for it - `transition`

Let's not make names shorter if we can